packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -325,15 +349,16 @@ export class PendingStateManager implements IDisposable {
 
 		for (const message of batch) {
 			const {
-				serializedOp: content,
+				runtimeOp,
 				referenceSequenceNumber,
 				localOpMetadata,
 				metadata: opMetadata,
 			} = message;
 			const pendingMessage: IPendingMessage = {
 				type: "message",
 				referenceSequenceNumber,
-				content,
+				content: serializeOp(runtimeOp),
```

Comment:
Is there a reason to hold both? I imagine this will double the memory usage for really large ops.